data/io.github.seadve.Kooha.metainfo.xml.in.in

```diff
@@ -22,12 +22,16 @@
   <screenshots>
     <screenshot type="default">
       <image>https://raw.githubusercontent.com/SeaDve/Kooha/main/data/screenshots/screenshot.png</image>
+      <caption>Recording options and button to start recording</caption>
     </screenshot>
     <screenshot>
       <image>https://raw.githubusercontent.com/SeaDve/Kooha/main/data/screenshots/screenshot2.png</image>
+      <caption>In-progress recording duration and button to stop recording</caption>
     </screenshot>
     <screenshot>
       <image>https://raw.githubusercontent.com/SeaDve/Kooha/main/data/screenshots/screenshot3.png</image>
+      <caption>Countdown to start recording and button to cancel recording</caption>
+      <cap
```

Comment:
The ci seems to be failing here

Reply:
Oopsie 😅

Reply:
I pushed a fix. Thanks for reviewing so quickly :)

Reply:
thankss!